go.mod

```diff
@@ -1,16 +1,18 @@
 module main
 
-go 1.20
+go 1.21
 
 require (
+	github.com/containerd/containerd v1.7.13 // indirect
 	github.com/exasol/error-reporting-go v0.2.0
 	github.com/exasol/exasol-test-setup-abstraction-server/go-client v0.3.5
 	github.com/gin-gonic/gin v1.9.1
 	github.com/gorilla/websocket v1.5.1
+	github.com/opencontainers/runc v1.1.12 // indirect
 	github.com/stretchr/testify v1.8.4
 	github.com/swaggo/files v1.0.1
 	github.com/swaggo/gin-swagger v1.6.0
-	github.com/swaggo/swag v1.16.2
+	github.com/swaggo/swag v1.16.3
```

Comment:
Please update the `swag` version also in the other places (search for `1.16.2`) to ensure consistent behavior.